test/test_loggers.py

```diff
@@ -7,9 +7,23 @@
 
 import pytest
 import torch
-import torchvision
+
+try:
+    import torchvision
+
+    _has_tv = True
+except ImportError:
+    _has_tv = False
+
+try:
+    import mlflow
+
+    _has_mlfow = True
+except ImportError:
+    _has_mlfow = False
+
 from torchrl.trainers.loggers.csv import CSVLogger
-from torchrl.trainers.loggers.mlflow import MLFlowLogger, _has_mlflow
+from torchrl.trainers.loggers.mlflow import MLFlowLogger
```

Comment:
if removine _has_mlflow here probably worth doing the same for _has_tb and _has_wandb

Reply:
I mean, to be consistent I would change also lines 27 and 28

Reply:
but how do we get `_has_tb` and `_has_wandb` then? Sorry I probably misunderstood